jSka/config/properties.py

```diff
@@ -0,0 +1,18 @@
+import os
+from jSka.core.exceptions import ImproperlyConfigured
+
+def get_env_variable(var_name):
+
+    try:
+        return os.environ[var_name]
+    except:
+        message = "Set the {} environment variable".format(var_name)
+        raise ImproperlyConfigured(message)
+
+
+INGEST_DIR=get_env_variable('JSKA_INGEST_DIR')
+ARCHIVE_DIR=get_env_variable('JSKA_ARCHIVE_DIR')
+
+NAME_COLUMN=get_env_variable('NAME_COLUMN')
```

Comment:
I'm curious about the driver for having these configurable via env vars.

Reply:
For INGEST_DIR and ARCHIVE_DIR, this is because on the "real" deployment server these locations are mounted volumes that the IT Dept manages. Locally for development its in my home dir some where. 

NAME_COLUMN is because the headers in the ingest files have been inconsistent.